ts/features/lollipop/hooks/usePublicKeyState.tsx

```diff
@@ -37,6 +40,6 @@ export const usePublicKeyState = () => {
           )()
       )
     );
-  }, [keyTag]);
+  }, [keyTag, publicKeyState.kind]);
```

Comment:
Why is this needed? The original dependency is the keyTag so this useEffect is not supposed to run but once

Reply:
Yeah, you are right! I put my log not inside the use effect.